lib/routes/home/floating_actions_bar.dart

```diff
@@ -142,14 +143,21 @@ class FloatingActionsBar extends StatelessWidget {
                                 ));
                                 return;
                               }
+                              if (!isValidNodeId(scannedString)) {
```

Comment:
Shouldn't it be the opposite condition?
```suggestion
                              if (isValidNodeId(scannedString)) {
```